components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/inbound/IdentityServlet.java

```diff
@@ -107,7 +108,11 @@ private HttpIdentityResponse process(HttpServletRequest request, HttpServletResp
             }
             return responseBuilder.build();
         } catch (FrameworkException e) {
-            log.error("Failed to process IdentityRequest", e);
+            if (e instanceof SessionExtenderClientException) {
```

Comment:
```suggestion
            if (e instanceof FrameworkClientException) {
```
Can we check for FrameworkClientException? So it will cover a wide area.